File: tests/rest-assured-java/src/test/java/com/portfolio/qa/core/Endpoints.java
package com.portfolio.qa.core;

public final class Endpoints {

    private Endpoints() {
    }

    public static final String HEALTH = "/health";
    public static final String DOCS = "/docs/";

    public static final String TEST_RESET = "/api/v1/test/reset";

    public static final String TASKS = "/api/v1/tasks";
    public static final String TASK_BY_ID = "/api/v1/tasks/{id}";
    public static final String TASK_STATUS_BY_ID = "/api/v1/tasks/{id}/status";
}

File: tests/karate-java/src/resources/karate/core/endpoints.js
function fn() {
  return {
    health: '/health',
    docs: '/docs/',
    testReset: '/api/v1/test/reset',
    tasks: '/api/v1/tasks',

    taskById: function (id) {
      return '/api/v1/tasks/' + id;
    },

    taskStatusById: function (id) {
      return '/api/v1/tasks/' + id + '/status';
    }
  };
}